src/Generators/Microsoft.Gen.AutoClient/Emitter.cs

```diff
@@ -412,10 +435,10 @@ private void EmitClassVariablesAndConstructor(RestApiType restApiType)
 
             if (restApiMethod.FormatParameters.Count == 0 && !restApiMethod.AllParameters.Any(p => p.IsQuery))
             {
-                OutLn(@$"private static readonly {Uri} _uri{requestName} = new(""{restApiMethod.Path}"", {UriKind}.Relative);");
+                OutLn(@$"public static readonly {Uri} Uri{requestName} = new(""{restApiMethod.Path}"", {UriKind}.Relative);");
             }
 
-            OutLn(@$"private static readonly {RequestMetadata} _requestMetadata{requestName} = new()");
+            OutLn(@$"public static readonly {RequestMetadata} RequestMetadata{requestName} = new()");
```

Comment:
Why is request metadata and Uri "public" now? You want users to be able to change those statically?

Reply:
All of the statics are not in a nested static class which is private, so customers still can't get to that stuff.

The reason to put all the statics into a class was to reduce "naming pollution" of the main class. The names in the nested types don't have any risk of conflicting with names of things in the main type being generated.

Reply:
Ohh... Got it. It's so hard to read code written by other code...
I think we should keep on the repo the source-generated file somehow... It would help a ton to review.

Thanks!